src/API/ReduxStore.js

```diff
@@ -0,0 +1,266 @@
+            blockNumber,
+            timestamp
+          }
+        }
+      }
+      console.log(newState.nodes)
+      return newState
+    case '[NODE]:LOCAL:UPDATE_SYNCING':
+      const { startingBlock, currentBlock, highestBlock } = action.payload
+      newState = {
+        ...state,
+        nodes: {
+          ...state.nodes,
+          local: {
+            ...state.nodes.local,
+            sync: {
+              ...state.nodes.local.sync,
+              startingBlock,
+              currentBlock,
+              highestBlock,
+              knownStates,
+              pulledStates
+            }
+          }
+        }
+      }
+      return newState
+    case '[NODE]:LOCAL:UPDATE_NETWORK':
+      const { network } = action.payload
+      newState = {
+        ...state,
+        nodes: {
+          ...state.nodes,
+          network
+        }
+      }
+      return newState
+    case '[NODE]:LOCAL:UPDATE_SYNC_MODE':
+      const { syncMode } = action.payload
+      newState = {
+        ...state,
+        nodes: {
+          ...state.nodes,
+          syncMode
+        }
+      }
+      return newState
+    case '[NODE]:LOCAL:UPDATE_PEER_COUNT':
+      const { connectedPeers } = action.payload
+      newState = {
+        ...state,
+        nodes: {
+          ...state.nodes,
+          local: {
+            ...state.nodes.local,
+            sync: {
+              ...state.nodes.local.sync,
+              connectedPeers
+            }
+          }
+        }
+      }
+      return newState
+    case 'SET_LOCAL_PEERCOUNT':
+      newState.nodes.local.sync.connectedPeers = action.payload
+      return newState
+    case 'SET_REMOTE_BLOCK': {
+      const blockHeader = action.payload
+      newState.nodes.remote.blockNumber = blockHeader.number
+      newState.nodes.remote.timestamp = blockHeader.timestamp
+      return newState
+    }
+    case 'ADD_TAB': {
+      const tab = action.payload
+      const tabs = [...state.tabs, tab]
+      newState.tabs = tabs
+      return newState
+    }
+    case 'ADD_ACCOUNT': {
+      const acc = action.payload
+      const accounts = [...state.accounts, acc]
+      newState.accounts = accounts
+      return newState
+    }
+    case 'SET_TX': {
+      const tx = action.payload
+      newState.newTx = tx
+      return newState
+    }
+    case 'EDIT_TAB': {
+      /*
+      const newState = Object.assign({}, state)
+      const tab = action.payload
+      const tabs = [...state.tabs] // copy tabs state
+      const tabIdx = tabs.findIndex(t => (t.id === tab.id)) // find changed item
+      const tabM = {
+        ...tabs[tabIdx], // create copy of changed item
+        icon: icon // & modify copy
+      }
+      tabs[tabIdx] = tabM // write changes to new tabs state
+      return { tabs }
+      */
+      break
+    }
+    default:
+      return state
+  }
+}
+/* eslint-enable */
+
+const store = createStore(mistApp, applyMiddleware(ReduxThunk))
+
+export default store
```

Comment:
This file was deleted when the new store (`src/store/index.js`) was created. Is redundant so can be deleted.